app/components/search_result/results_metadata_component.html.erb

```diff
@@ -0,0 +1,55 @@
+<dl class="dl-horizontal dl-invert results-metadata-section col-md-8">
+    <% if present_fields['extent'] %>
+      <dt><%= document.extent_label %></dt>
+      <dd><%= document.extent %></dd>
+    <% end %>
+
+    <% if document.is_a_database? && present_fields['db_az_subject'] %>
+        <dt>Database topics</dt>
+        <dd>
+            <%= safe_join(document[:db_az_subject].map { |subject| helpers.link_to_database_search(subject) }, '; ') %>
```

Comment:
There were two ways this was rendered in the old templates. This way and the way below. From what I can tell they do the same thing but if whoever is reviewing the PR could confirm that it would be appreciated.
```
<%= document[:db_az_subject].map do |subject| %>
            <% link_to(subject, search_catalog_path(f: {db_az_subject: [subject], document.format_key => ['Database']})) %>
          <% end.join('; ').html_safe %>
```